lib/ar/sequence/schema_dumper.rb

```diff
@@ -36,18 +36,31 @@ def sequences(stream)
           next unless @connection.custom_sequence?(sequence_full_name)
 
           start_value = seq["start_value"]
+          minimum_value = seq["minimum_value"]
+          maximum_value = seq["maximum_value"]
+          cycle_option = seq["cycle_option"]
           increment = seq["increment"]
 
           options = []
 
-          if start_value && Integer(start_value) != 1
+          if start_value && Integer(start_value) != Integer(minimum_value)
             options << "start: #{start_value}"
           end
 
+          if minimum_value && Integer(minimum_value) != 1
+            options << "min: #{minimum_value}"
+          end
+
+          if maximum_value && Integer(maximum_value) != (2**63 - 1)
```

Comment:
Technically this maximum value only applies to _ascending_ (default) sequences with `bigint` (default) type, so if we want to add support for descending sequences or non-bigint sequences, we would need to correct this logic.

Presumably we could use `information_schema.sequences.data_type` to check the max (or do some math based on `information_schema.sequences.numeric_precision` maybe?) and check that `increment` is negative to detect descending sequences, but I'm not even sure why you would use either of those features, and they would be a separate PR regardless.